README.md

```diff
@@ -42,16 +42,14 @@ npx lerna bootstrap
 
 ## Releasing new versions
 
-The releases are currently manually generated. Make sure you have the latest
-`master` branch of this repository, and are in the root folder when running:
-
-```bash
-git checkout master && git pull
-```
-
-To automatically increase the versions numbers for each project based on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
-and release them, run:
-
-```bash
-npx lerna publish
-```
+Release are generated automatically when Pull Requests are merged into the `master` branch.
+
+This process is automated thanks to [conventional
+changelog](https://github.com/conventional-changelog/conventional-changelog)
+style commit messages e.g. `docs(readme): fixed spelling mistake` and
+[`semantic-release`](https://github.com/semantic-release/semantic-release).
```

Comment:
Is this correct? Does Lerna use `semantic-release` under the hood? Or is Lerna a replacement for `semantic-release` in this repo?